File: GenshinAcademy.ArtiRater/GenshinAcademy.ArtiRater.Web/Models/Character.cs
namespace GenshinAcademy.ArtiRater.Web.Models
{
    public class Character
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Element { get; set; }
        
        public Uri IconUrl { get; set; }

        public IReadOnlyCollection<ArtifactStatsProfit>? StatsProfit { get; set; }
    }
}

File: internal/ferret/web_models/character.go
package web_models

type FerretCharacter struct {
	CharacterId string        `json:"character_id"`
	Name        string        `json:"name"`
	Element     string        `json:"element"`
	IconUrl     string        `json:"icon_url"`
	StatsProfit []StatsProfit `json:"stats_profit"`
}
